packages/vitest/LICENSE.md

```diff
@@ -372,6 +372,23 @@ Repository: git@github.com:moxystudio/node-cross-spawn.git
 
 ---------------------------------------
 
+## detect-indent
```

Comment:
Do you know why there is this diff in the LICENSE.md? Maybe we could merge this in another PR?

Reply:
some merge from main...

Reply:
Because it only generate on build script, so someone add a dep but not running the build will missing the changes for licneses. Not a big deal tho.